releasenotes/notes/tomography-b091ce13d6983bc1.yaml

```diff
@@ -5,6 +5,16 @@ features:
     :class:`~.StateTomography` and :class:`~.ProcessTomography` experiments.
     These allow specifying intended backend, a custom analysis class, or a
     custom target for fidelity calculations, when initializing the experiments.
+  - |
+    Adds conditional fragment tomography support to the
+    :class:`.StateTomography` and :class:`.ProcessTomography` experiments.
+    This allows either conditioning on classical bit values in the tomography
```

Comment:
To me this this the definition of conditional, and its the same as with conditional gates: a conditional gate is a gate conditioned on a classical bit value, conditional tomography is a tomographic reconstruction conditioned on a classical bit value.

As far as tomography is concerned a dynamic circuit is no different from any other circuit. If it has classical bits, and you want to condition your reconstruction on them, you can.

Reply:
> conditional tomography is a tomographic reconstruction conditioned on a classical bit value.

I like this description. This makes definition clear. Otherwise someone like me before might think this is a tomography for conditional gates.